articles/tutorials/04_working_with_textures/index.md

```diff
@@ -0,0 +1,364 @@
+---
+title: "Chapter 04: Working with Textures"
+description: In this chapter, we'll compare loading a texture from file versus pre-processing it using the content pipeline and how to render the textures as visuals in your game.
+---
+
+Textures are images that you use in your game to represent the visual graphics to the player. In order to use them, you need to either create the texture in code or load an existing one from a file and then draw the texture using the `SpriteBatch`.
+
+## Loading a Texture
+
+To load a texture in your game, you can either load it directly from its file (*.png*, *.jpg*, etc) or you can pre-process it using the *content pipeline* and load it using the `ContentManager`.

```

Comment:
Maybe explain what is the difference. The developer sees it can load it directly from a file, then he knows he can pre-process it to "load it" again? What's the benefit of using the content manager? I noticed it's explained way below. Maybe pull it up?